common/constants/shared.ts

```diff
@@ -52,9 +52,9 @@ export const observabilityPanelsID = 'observability-dashboards';
 export const observabilityPanelsTitle = 'Dashboards';
 export const observabilityPanelsPluginOrder = 5095;
 
-export const observabilityIntegrationsID = 'observability-integrations';
+export const observabilityIntegrationsID = 'integrations';
```

Comment:
Split for isolation

Reply:
Renamed, but I think it still makes sense to keep in the same file